### ISSUE TITLE
How can I customize objective function for regression solution

### ISSUE THREAD

**Issue by emigmo** [dmlc/xgboost#1192]
in classification there are:
       def logregobj(preds, dtrain):
and def evalerror(preds, dtrain):
for the classification
 can I customize loss function for regression ?

**Comment by khotilov** [comment]
What makes you think that you cannot do that for regression?
